evalml/tests/model_understanding_tests/prediction_explanations_tests/test_algorithms.py

```diff
@@ -105,9 +101,6 @@ def test_shap(estimator, problem_type, n_points_to_explain, X_y_binary, X_y_mult
     if problem_type not in estimator.supported_problem_types:
         pytest.skip("Skipping because estimator and pipeline are not compatible.")
 
-    if problem_type == ProblemTypes.MULTICLASS and estimator.model_family == ModelFamily.CATBOOST:
-        pytest.skip("Skipping Catboost for multiclass problems.")
```

Comment:
After deleting this check, this test does verify that multiclass catboost works. I verified this by changing the multiclass catboost impl to error, and saw that this test errored out.